.github/workflows/main.yml

```diff
@@ -190,7 +190,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest]
-        version: ['v2.7.6', 'v2.8.5', 'v2.9.4', 'v2.10.5', 'v2.11.6', 'nightly']
+        version: ['v2.7.6', 'v2.8.5', 'v2.9.4', 'v2.10.5', 'v2.11.5', 'nightly']
```

Comment:
Mistake? Also. below.

Reply:
This is for testing - I was testing it in my fork - hence a lot of rebasing. Sorry for all the noise its creating for you 😓